test/Datadog.Trace.TestHelpers/Datadog.Trace.TestHelpers.csproj

```diff
@@ -1,8 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <ItemGroup>
-    <PackageReference Include="MessagePack" Version="1.9.3" />
     <PackageReference Include="System.Collections.Immutable" Version="1.5.0" />
+    <PackageReference Include="MsgPack.Cli" Version="1.0.1" />
```

Comment:
This test uses `MsgPack.Cli` to _deserialize_ spans. I'll leave the task of refactoring the test for later.